artifacts/openapi.yaml

```diff
@@ -278,12 +278,19 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Capture'
-        devices:
+        interfaces:
           description: |-
-            The emulated device settings that will be configured on the traffic generator.
+            The emulated interfce settings that will be configured on the traffic generator.
+            Each of this defines one Ethernet, Loopback Ipv4 or Loopback Ipv6 interfaces.
```

Comment:
Each defines a Ethernet, a Loopback IPv4 or a Loopback IPv6 interface.

Reply:
New PR#154 has interfaces and protocol configurations within each device and description is updated accordingly.